### DISCUSSION TITLE
Composite key on id

### DISCUSSION THREAD

**Discussion by GuilhermeHirsch** [supabase/supabase#2243] [Questions]
Hey guys! Is a composite key on id automatically generated?
I noticed this behavior when importing the table into Budibase.

![124606809-25aba080-de65-11eb-974b-355999a736b2](https://user-images.githubusercontent.com/33108596/124624331-2cc9b300-de53-11eb-8339-8c5b5ade8591.png)

**Comment by burggraf** [comment]
Hi GuilhermeHirsch,

I don't think a composite key is automatically created anywhere.  Have you been able to figure out where these multiple id fields are coming from?
